iothub/service/src/IoTHubExceptionResult.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// A class used as a model to deserialize error response object sent by IoT Hub
```

Comment:
nit: A class used as a model to deserialize error response object ~sent by IoT Hub~ received from IoT Hub